libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -1420,23 +1467,28 @@ def update_state(
                     **checkpoint_metadata,
                     "source": "update",
                     "step": step + 1,
-                    "writes": {as_node: values},
+                    "writes": {k: v for k, v in valid_updates},
```

Comment:
could we use better names for unpacking here
```suggestion
                    "writes": {as_node: values for as_node, values in valid_updates},
```